pyproject.toml

```diff
@@ -6,7 +6,7 @@ build-backend = "hatchling.build"
 name = "maximum-independent-set"
 description = "A Python library designed to help users design classical- and quantum-driven solutions for the Maximum Independent Set (MIS) problem."
 readme = "README.md"
-version = "0.2.1"
+version = "0.2.2"
 requires-python = ">=3.10,<3.13"
```

Comment:
Is it necessary to limit to <3.13?
3.13 came out 11 months ago.

Reply:
Just checked: we depend on a version of qoolqit that doesn't support 3.13.

Sigh...

I'll try again once #113 has merged.